applications.yaml

```diff
@@ -1,11 +1,26 @@
 ---
+- name: AhkWindowSpy
+  identifier:
+    kind: title
+    id: Window Spy
+  float_identifiers:
+    - kind: title
+      id: Window Spy
+      comment: "Target AutoHotKey's Window Spy pop-up"
```

Comment:
Let's omit the surrounding double quotes here

Reply:
Fixed!